deployment/README.md

```diff
@@ -98,20 +98,14 @@ An invocation supplying a properly signed Kibana cert might be:
 ## Create Supporting ServiceAccounts
 
 The deployer must run under a service account defined as follows:
+(Note: change `:logging:` below to match the project name.)
 
-    $ oc create -f - <<API
-    apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: logging-deployer
-    secrets:
-    - name: logging-deployer
-    API
-
-    $ oc policy add-role-to-user edit \
+    $ oc process -f deployer-account.yaml | oc create -f -
+    $ oc policy add-role-to-user edit --serviceaccount logging-deployer
+    $ oc policy add-role-to-user daemonset-admin --serviceaccount logging-deployer
```

Comment:
No, it works this way. I wanted the role-binding to be scoped to the project. The oauthclient permission can't be scoped to a project, but this can. I was trying to get it working with local policy but hit a snag and this worked.